packages/pulpcore/python-pulpcore/python-pulpcore.spec

```diff
@@ -20,8 +20,6 @@ BuildRequires:  python3-aiodns
 BuildRequires:  python3-aiofiles
 BuildRequires:  python3-aiohttp
 BuildRequires:  python3-backoff
-Requires:       python3-coreapi >= 2.3.3
```

Comment:
Will you create a separate PR to drop coreapi?

Reply:
no, because it's still pulled in via python-drf-yasg :(